e2e/specs/register-new-patient.spec.ts

```diff
@@ -31,29 +30,33 @@ test('Register a new patient', async ({ page, api }) => {
     await patientRegistrationPage.waitUntilTheFormIsLoaded();
   });
 
-  await test.step('And then I click on fill new values into the registration form and then click the `Submit` button', async () => {
+  await test.step('And then I fill the registration form and then click the `Submit` button', async () => {
     await patientRegistrationPage.fillPatientRegistrationForm(formValues);
   });
 
   await test.step("Then I should be redirected to the new patient's chart page and a new patient record should be created from the information captured in the form", async () => {
-    await expect(page).toHaveURL(new RegExp('^[\\w\\d:\\/.-]+\\/patient\\/[\\w\\d-]+\\/chart\\/.*$'));
-    const patientUuid = /patient\/(.+)\/chart/.exec(page.url())?.[1] ?? null;
-    expect(patientUuid).not.toBeNull();
-
-    const patient = await getPatient(api, patientUuid);
-    const { person } = patient;
-    const { givenName, middleName, familyName, sex } = formValues;
-
-    expect(person.display).toBe(`${givenName} ${middleName} ${familyName}`);
-    expect(person.gender).toMatch(new RegExp(sex[0], 'i'));
-    expect(dayjs(person.birthdate).format('DD/MM/YYYY')).toBe(
-      `${formValues.birthdate.day}/${formValues.birthdate.month}/${formValues.birthdate.year}`,
-    );
-    expect(person.preferredAddress.address1).toBe(formValues.address1);
-    expect(person.preferredAddress.cityVillage).toBe(formValues.cityVillage);
-    expect(person.preferredAddress.stateProvince).toBe(formValues.stateProvince);
-    expect(person.preferredAddress.country).toBe(formValues.country);
-    expect(person.attributes[0].display).toBe(formValues.phone);
+    const patientChartUrlRegex = new RegExp('^[\\w\\d:\\/.-]+\\/patient\\/[\\w\\d-]+\\/chart\\/.*$');
+    await page.waitForURL(patientChartUrlRegex);
+    await expect(page).toHaveURL(patientChartUrlRegex);
+    await page.locator('header[data-openmrs-role="patient banner"]').waitFor();
+    await expect(page.locator('header[data-openmrs-role="patient banner"]')).toBeVisible();
+  });
+
+  await test.step("And I should the newly registered patient's details in the patient banner", async () => {
+    await expect(page.getByText(/johnny donny ronny/i)).toBeVisible();
```

Comment:
I believe it would be more effective to make the names case-sensitive. Same with the addresses. What do you think?
```suggestion
    await expect(page.getByText("Johnny Donny Ronny")).toBeVisible();
```

Reply:
shouldn't it be` page.locator('[data-openmrs-role="patient banner"]')`?

Reply:
Regarding waitFor:  https://playwright.dev/docs/testing-library#replacing-waitfor

Reply:
> shouldn't it be` page.locator('[data-openmrs-role="patient banner"]')`?

No luck